fix multiple WSL issues

barangerbenjamin opened the pull request:
- Moved `export DISPLAY=:0` after the dotfiles setup since the script will change the content of the `.zshrc` file. 

- Change the installtion of nodejs, the default install will install the `v4.2.6` and students will get an error when `rails new app_name --webpack`. Fixes #67 

- Changed all mention of `apt-get` to simple `apt`. Fixes #68 

- Add the installation of the graphical GTK component needed for Xming to work properly. Fixes #71

db0sch commented:
@barangerbenjamin You should not change the file `WINDOWS_LINUX_SUBSYSTEM.md`.
This file is generated by the `ruby build.rb` command.
You need to change the code into the partials.

barangerbenjamin commented:
@db0sch Yes silly me!
Will change that 🙏

barangerbenjamin commented:
I've updated the partials for WSL
The build for WSL was using a ubuntu_oh_my_zsh file so i've created it's own wsl_oh_my_zsh.md since the installation differs now.
I had to update the`build.rb` in consequence.
Also updated some Ubuntu partials to remove all the `apt-get` to replace them with simple `apt`
